### DISCUSSION TITLE
Who had the best 80s hair and why is it Kurt Russell?

### DISCUSSION THREAD

**Discussion by ladylazarus3** [rbms-bsc/DCRMR#146] [Polls]
This poll is simply to make the new Discussions space more welcoming and less empty.  Don't let the silliness of this post scandalize, i'm sure it will self-destruct very shortly.

**Comment by eblakedc** [comment]
Dolly Parton's 80s hair was definitely better than Kurt Russell's, but the _best_ 80s hair was worn by Maria Nikolidakis, who sat in front of me in grade 12 English, and completely blocked my view of the board with her massive perm and back-combed hair-sprayed bangs.

**Reply by ladylazarus3** [reply]
Important update:  Dolly is currently in second place only behind Maria Nikolidakis!  <3  (My sympathies, i would have been sorely tempted to bring a pair of my mom's beautician scissors to class... or a flat iron. >;)   )  

A "large volume" thank you to everyone engaging in and gleefully adopting our new GitHub discussions feature. <3  Certainly, it has brought joy and smiles into my day.  <3  The first sparkles of many conversations to come.

k
